pyproject.toml

```diff
@@ -37,7 +37,7 @@ docs = [
     "mike ~=2.0",
     "mkdocs ~=1.5",
     "mkdocs-awesome-pages-plugin ~=2.9",
-    "mkdocs-material ~=9.2",
+    "mkdocs-material >=9.5.5",
```

Comment:
This is fine. But for generation of documentation, I think we could consider to freezing the requirements.

Reply:
I think that it is good to continue to get these notifications on newly discovered vulnerabilities, also for the documentation..?

Reply:
Good point

Reply:
Let's make another issue if wee want to freeze the documentation requirements.